### DISCUSSION TITLE
Create official websites for Sideband/Nomad Network.

### DISCUSSION THREAD

**Discussion by hardBSDk** [markqvist/Reticulum#131] [Ideas]
Would be nice if you create websites for these Reticulum projects, like the [Reticulum website](https://reticulum.network/).

**Comment by markqvist** [comment]
Totally agree with this. A few simple, static pages would probably be sufficient for now.

**Reply by hardBSDk** [reply]
Maybe with GitHub Pages for some features and free hosting.

You could implement some real-time stats on the website (JS?), like Matrix (homeserver bubbles) and [mempool.space](https://mempool.space).

**Reply by markqvist** [reply]
That is cool, but I think I'd rather like to keep it simple for now :)

**Comment by markqvist** [comment]
I just refreshed my site over at [unsigned.io](https://unsigned.io). Both Sideband and Nomad Net have their own pages there now:

- [Sideband](https://unsigned.io/sideband/)
- [Nomad Network](https://unsigned.io/nomadnet/)

I think that will be sufficient for now :)

**Reply by hardBSDk** [reply]
Provide the source code, I want to translate them.

**Reply by markqvist** [reply]
Thanks! Here are the markdown sources:

[Nomad_Network.md](https://github.com/markqvist/Reticulum/files/10207627/Nomad_Network.md)
[Sideband.md](https://github.com/markqvist/Reticulum/files/10207628/Sideband.md)

**Comment by Swissbandit** [comment]
[Nomad_Network_de.md](https://github.com/markqvist/Reticulum/files/10251932/Nomad_Network_de.md)
[Sideband_de.md](https://github.com/markqvist/Reticulum/files/10251933/Sideband_de.md)
 Here are the two Files, Translated in German

**Reply by markqvist** [reply]
Thank you very much @Swissbandit! Now I just gotta add some multilanguage support to unsigned.io ;)
